server/views/pages/plan-history.njk

```diff
@@ -29,15 +29,18 @@
 {% endblock %}
 
 {% block content %}
-    <p class='govuk-!-margin-top-5'>{{ locale.page.introduction }}</p>
+    {% if data.notes.length == 0 %}
+        <p class='plan-history-intro govuk-!-margin-top-5'>{{ locale.page.noPlanIntroduction }}</p>
```

Comment:
if you move the if/endif to inside the <p> then you only need that line once rather than twice. Can you do that please? So code will look like
p
if
else
endif
p